static/js/settings_org.js

```diff
@@ -320,18 +320,14 @@ function _set_up() {
         _.each(property_types.settings, function (v, k) {
             property_type_status_element(k).hide();
         });
-        var name_status = $("#admin-realm-name-status").expectOne();
-        var waiting_period_threshold_status = $("#admin-realm-waiting-period-threshold-status").expectOne();
-        name_status.hide();
-        waiting_period_threshold_status.hide();
+        var status = $(".settings-section.show .alert").eq(0);
```

Comment:
These two functions don't do the same thing, and I don't think it matters whether one exists or five exists. It just grabs the first alert and uses it for generic UI alerts.